ush/wave_prnc_ice.sh

```diff
@@ -84,45 +76,26 @@
 
   if [ -f ice.grib ]
   then
-    set +x
     echo "   ice.grib copied ($file)."
-    set_trace
   else
-    set +x
-    echo ' '
-    echo '************************************** '
-    echo "*** FATAL ERROR: NO ICE FILE $file ***  "
-    echo '************************************** '
-    echo ' '
-    set_trace
-    echo "FATAL ERROR - NO ICE FILE (GFS GRIB)"
+    echo "ERROR: NO ICE FILE $file"
```

Comment:
Why not a FATAL ERROR? Apologies if this was discussed already in other comments.